includes/admin/reporting/export/class-batch-export-taxed-orders.php

```diff
@@ -98,7 +98,7 @@ public function get_data() {
 		);
 
 		if ( ! empty( $this->start ) || ! empty( $this->end ) ) {
-			$args['date_query'] = $this->get_date_query();
+			$args['date_created_query'] = $this->get_date_query();
```

Comment:
I think this should be applied to the `get_percentage_complete` method in this class too, the same way you did for file downloads?